Index: linux-4.0/kernel/reboot.c
===================================================================
--- linux-4.0.orig/kernel/reboot.c
+++ linux-4.0/kernel/reboot.c
@@ -217,9 +217,9 @@ void kernel_restart(char *cmd)
 	migrate_to_reboot_cpu();
 	syscore_shutdown();
 	if (!cmd)
-		pr_emerg("Restarting system\n");
+		pr_info("Restarting system\n");
 	else
-		pr_emerg("Restarting system with command '%s'\n", cmd);
+		pr_info("Restarting system with command '%s'\n", cmd);
 	kmsg_dump(KMSG_DUMP_RESTART);
 	machine_restart(cmd);
 }
